src/components/test/Panel-test.js

```diff
@@ -0,0 +1,68 @@
+import { mount } from 'enzyme';
+
+import { Panel } from '../Panel';
+import { $imports } from '../Panel';
+
+import { checkAccessibility } from '../../../test/util/accessibility';
+import mockImportedComponents from '../../../test/util/mock-imported-components';
+
+describe('Panel', () => {
+  const createPanel = props =>
+    mount(
+      <Panel title="Test Panel" {...props}>
+        Test panel
+      </Panel>
+    );
+
+  beforeEach(() => {
+    $imports.$mock(mockImportedComponents());
+  });
+
+  afterEach(() => {
+    $imports.$restore();
+  });
+
+  it('renders the provided title', () => {
+    const wrapper = createPanel({ title: 'My Panel' });
+    const titleEl = wrapper.find('.Panel__title');
+    assert.equal(titleEl.text(), 'My Panel');
+  });
+
+  it('renders an icon if provided', () => {
+    const wrapper = createPanel({ icon: 'restricted' });
+
+    const icon = wrapper.find('SvgIcon').filter({ name: 'restricted' });
+
+    assert.isTrue(icon.exists());
```

Comment:
I've gone ahead and made this change as suggested here because it's more consistent with the other tests in this module, _but_ I wanted to explain the motivation in the first place for the spacing.

Our unit-test conventions suggest that we structure a given unit test in three "sections":

1. Code that gets the test case ready (in this case, creating the wrapper)
2. "go/execute" (in this case querying for the component/prop combination we'd expect)
3. Assertions (that thing we looked for exists, here)

In a test this simple, keeping it all tight is fine, but in more complex tests this explicit separation can be helpful.

Reply:
It is very useful to know. Thanks for the explanation.